scripts/generate-clients.mjs

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env zx
+import "zx/globals";
+import * as k from "kinobi";
+import { rootNodeFromAnchor } from "@kinobi-so/nodes-from-anchor";
+import { renderVisitor as renderJavaScriptVisitor } from "@kinobi-so/renderers-js";
+import { renderVisitor as renderRustVisitor } from "@kinobi-so/renderers-rust";
+import { getAllProgramIdls } from "./utils.mjs";
+
+// Instanciate Kinobi.
+const [idl, ...additionalIdls] = getAllProgramIdls().map(idl => rootNodeFromAnchor(require(idl)))
+const kinobi = k.createFromRoot(idl, additionalIdls);
+
+// Update programs.
+kinobi.update(
+  k.updateProgramsVisitor({
+    "stakeProgram": { name: "stake" },
+  })
+);
+
+// Update accounts.
+kinobi.update(
+  k.updateAccountsVisitor({
+    counter: {
+      seeds: [
+        k.constantPdaSeedNodeFromString("utf8", "counter"),
+        k.variablePdaSeedNode(
+          "authority",
+          k.publicKeyTypeNode(),
+          "The authority of the counter account"
+        ),
+      ],
+    },
+  })
+);
```

Comment:
I think all of these "counter" configs can get dropped, since they're not for the stake program. If we need any configs like these for the program, we can add them later!

Reply:
My plan is to fix all the client generation in a separate PR, including adding a basic Kinobi configuration.